learning_observer/learning_observer/auth/events.py

```diff
@@ -230,8 +234,12 @@ async def chromebook_auth(request, headers, first_event, source):
     if untrusted_google_id is None:
         return False
 
+    payload_for_validation = authdata.get('chrome_identity', {})
+    
+    auth_response = authenticate_payload(payload_for_validation)
```

Comment:
What about other forms of auth that we use? Those should also be authenticated.

Reply:
For this, @DrLynch and @pmitros agreed that we are authenticating with just the `chromebook_auth`  auth method. My implementation is isolated for that method and does not interfare with other methods' normal working conditions.